data/core/chitchat.md

```diff
@@ -543,9 +559,9 @@
     - slot{"onboarding": true}
     - utter_built_bot_before
 * deny
-    - utter_explain_stack
-    - utter_stack_details
-    - utter_explain_nlucore
+    - utter_explain_rasa_components
+    - utter_rasa_components_details
+    - utter_explain_nlucorex
 * how_to_get_started{"product": "nlu"}
```

Comment:
Is [https://github.com/RasaHQ/rasa-demo/pull/329/files#diff-889de434f9c9d30de56babe825a22b78R328](here) the only place where we deal with if someone says "i want to learn about rasa"?